### PR TITLE
Add updated Italian translation

### LINKED ISSUE #39: Update Italian translation for version 2.0
Version `1.2` of **vrms-rpm** had an Italian translation. Given the major overhaul of the program for version `2.0`, this translation can no longer be used. It'd be awesome if we could update the translation (or create an entirely new one) for the new version.

Files containing the old translation:
- `lang/outdated/it.po`
- `man/outdated/it.man`

New text (in English) to be translated:
- `lang/en.po`
- `man/en.man`

### PR DESCRIPTION
**Pull request by platipo**
Also removed outdated one. Fix #39.

### CODE REVIEW

File: lang/it.po [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+msgstr "    Mostra una ASCII-art di rms quando non esiste nessun pacchetto libero\n"
+       "    o quando i pacchetti non liberi sono più del 10%% del totale."
+
+msgid "HELP_OPTION_COLOUR\n"
+msgstr "    Controlla se le sequenze di escaping del teminale debbano essere utilizzate\n"
+       "    per colorare l'output. Il valore predefinito è 'auto', che colora l'output\n"
+       "    se si stampa a terminale, ma non su file o pipe.\n"
+
+msgid "HELP_OPTION_DESCRIBE\n"
+msgstr "    Oltre alla lista dei pacchetti, mostra il sommario per ognuno (breve descrizione)."
```
[the remark this answers is not in the document]
Comment (platipo) [reply lang/it.po]: The lines should wrap exactly less than 80 characters or about 80 characters?

File: lang/it.po [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+msgid "RMS_DISAPPOINTED\n"
+msgstr "Oltre 10%% di pacchetti non-liberi. Non ti piace la libertà\n"
+
+msgid "HELP_USAGE\n"
+msgstr "Uso: vrms-rpm [opzioni]\n"
+
+msgid "HELP_OPTION_ASCII\n"
+msgstr "    Mostra una ASCII-art di rms quando non esiste nessun pacchetto libero\n"
+       "    o quando i pacchetti non liberi sono più del 10%% del totale."
```
[the remark this answers is not in the document]
Comment (platipo) [reply lang/it.po]: Unfortunately [travis doesn't agree](https://travis-ci.org/suve/vrms-rpm/builds/440083106?utm_source=github_status&utm_medium=notification)

**Comment by platipo** [comment]
> The lines should wrap exactly less than 80 characters or about 80 characters?

**Comment by suve** [comment]
The lines of the messages should have no more than 80 characters.
```
msgstr "This is a long, long message\n"
```
There can be no more than 80 characters between the opening quote (`"`) and the linefeed+closing quote (`\n"`).

**Comment by platipo** [comment]
I rechecked all lines and it's good to go

**Comment by platipo** [comment]
Should I squash?

**Pull request merged**